stable/kubernetes-dashboard/values.yaml

```diff
@@ -0,0 +1,19 @@
+# Default values for kubernetes-dashboard
+# This is a YAML-formatted file.
+# Declare name/value pairs to be passed into your templates.
+# name: value
+
+image: gcr.io/google_containers/kubernetes-dashboard-amd64
+imageTag: "v1.6.0"
+imagePullPolicy: "IfNotPresent"
+
+namespace: kube-system
+
+nodeSelector: {}
+
+serviceType: ClusterIP
+
+resources:
+  requests:
+    memory: 512Mi
+    cpu: 512m
```

Comment:
Minor nit: required resources are high given what dashboard typically needs. Here are the defaults from GKE, for example:

```
        resources:
          limits:
            cpu: 100m
            memory: 50Mi
          requests:
            cpu: 100m
            memory: 50Mi
```
Not blocking for this PR, just something to consider.